src/dune_rules/coq/coq_stanza.ml

```diff
@@ -259,7 +261,8 @@ module Theory = struct
          Ordered_set_lang.Unexpanded.field
            "coqdoc_flags"
            ~check:(Dune_lang.Syntax.since coq_syntax (0, 8))
-       in
+       and+ coqdoc_header = field_o "coqdoc_header" String_with_vars.decode
+       and+ coqdoc_footer = field_o "coqdoc_footer" String_with_vars.decode in
```

Comment:
Here is how to guard the field
```suggestion
       and+ coqdoc_header =
         field_o
           "coqdoc_header"
           (Dune_lang.Syntax.since coq_syntax (0, 10) >>> String_with_vars.decode)
```

Reply:
My editor borked the line numbers, but you just have to put the check with a `>>>` which is monadic `;` before the decoder inside the field.

Reply:
Thanks.